pkg/exec/action.go

```diff
@@ -99,6 +99,7 @@ type Step struct {
 type Instruction struct {
 	Description     string        `yaml:"description"`
 	Command         string        `yaml:"command"`
+	WorkingDir      string        `yaml:"dir"`
```

Comment:
Since dir is optional, let's use `omitempty` here. This means that when dir is an empty string, it is not included in the yaml when marshaled to a string.

```suggestion
	WorkingDir      string        `yaml:"dir,omitempty"`
```

Reply:
yeah. Makes sense, obviously.